src/pages/notAuthorized/index.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+import { Result, Icon } from 'antd'
+
+import { FormButton } from '../login/styles'
+
+const NotAuthorized = ({ history }) => {
+  return (
+    <div style={{ padding: '40px 0' }}>
+      <Result
+        style={{ zoom: '1.1' }}
```

Comment:
whats the purpose of zoom here? why not setting the correct values for width & height of elements?